webapp/src/components/gitlab_issue_selector.tsx

```diff
@@ -75,7 +75,7 @@ const GitlabIssueSelector = ({name, required, theme, onChange, error, value, add
                 let prefix = '';
                 // Extract "username/projectName" from the issueURL parts
                 if (projectParts.length >= 5) {
-                    prefix = `${projectParts[projectParts.length - 5]}/${projectParts[projectParts.length - 4]}`;
+                    prefix = `${projectParts[3]}/${projectParts[4]}`;
```

Comment:
How is projectParts.length - 5 is 3 and -4 is 4 ?
Is accessing the 3rd and 4th index solves your issue ?
``` Is it possible that the URL doesn't match what want to to here? Maybe there are somtimes other suffixes on the URL that make this length - 4 etc. thing not work properly?  ``` 
According to this comment there could be other suffixes as well and directly accessing 3rd and 4th index may not solves the issue/ Please explain

Reply:
If this is of length 8 why are you checking for an array length equal to and greater than 5?